packages/table/src/utils/getTestUtils/getTestUtils.tsx

```diff
@@ -0,0 +1,112 @@
+import { getByLgId, queryBySelector } from '@lg-tools/test-harnesses';
+
+import { LGIDS } from '../../constants';
+
+import { TestUtilsReturnType } from './getTestUtils.types';
+
+export const getTestUtils = (
+  lgId: string = LGIDS.root,
+): TestUtilsReturnType => {
+  /**
+   * Queries the DOM for the element using the `data-lgid` data attribute.
+   * Will throw if no element is found.
+   */
+  const element: HTMLTableElement = getByLgId!(lgId);
+
+  const getAllHeaders = (): Array<HTMLTableCellElement> => {
+    const allHeaders = element.querySelectorAll<HTMLTableCellElement>(
+      `[data-lgid=${LGIDS.header}]`,
+    );
+
+    if (!allHeaders.length)
+      throw new Error('Unable to find any <th> elements.');
+
+    return Array.from(allHeaders);
+  };
+
+  const getHeaderByIndex = (index: number) => {
+    const allHeader = getAllHeaders();
+    const header = allHeader[index];
+    if (!header) return null;
+
+    const sortIcon = () =>
+      queryBySelector<HTMLButtonElement>(
+        header,
+        `[data-lgid=${LGIDS.sortIcon}]`,
+      );
+
+    return {
+      getElement: () => header,
+      getSortIcon: () => sortIcon(),
+    };
+  };
+
+  const getSelectAllCheckbox = () => {
+    const checkbox = queryBySelector<HTMLInputElement>(
```

Comment:
what happens if there is no checkbox?

Reply:
queryBySelector returns null if nothing is found. I have a test for this.